lib/engine/game/g_18_carolinas/game.rb

```diff
@@ -6,6 +6,8 @@
 require_relative 'map'
 require_relative 'step/buy_sell_par_shares_companies'
 require_relative 'step/track'
```

Comment:
i don't think you need any of these requires do you?

Reply:
I wasn't sure. I'll delete.